src/worker/tasks/sendTransactionWorker.ts

```diff
@@ -95,9 +99,48 @@ const handler: Processor<any, void, string> = async (job: Job<string>) => {
 
   if (transaction.status === "queued") {
     if (transaction.isUserOp) {
-      resultTransaction = await _sendUserOp(job, transaction);
+      // this can either be a regular backend wallet userop or a smart backend wallet userop
+      const accountAddress = transaction.accountAddress;
+      assert(accountAddress, "Invalid userOp parameters: accountAddress");
+
+      let adminAccount: Account | undefined;
+
+      try {
+        adminAccount = await getSmartBackendWalletAdminAccount({
+          accountAddress,
+          chainId: transaction.chainId,
+        });
+      } catch {
+        // do nothing, this might still be a regular backend wallet userop
+      }
+
+      try {
+        adminAccount = await getAccount({
+          chainId: transaction.chainId,
+          from: transaction.from,
+        });
+      } catch {
+        job.log(
```

Comment:
Don't need this added logging. When this job errors, the `.on("failed")` listener at the bottom of this file already `job.log`'s all errors with the error message.